examples/tree_3d_dgsem/elixir_mhd_dynamo.jl

```diff
@@ -0,0 +1,165 @@
+# A turbulent helically driven dynamo model in resistive compressible MHD.
+# We start with a weak small-scale magnetic field provided as Gaussian noise.
+# The energy is supplied via a forcing function f_force(x, t), which is delta-correlated in time.
+# For a comparison look into the Pencil Code example https://github.com/pencil-code/pencil-code/tree/master/samples/helical-MHDturb.
```

Comment:
Since we talk about the entire model (equations, initial condition and forcing) I prefer to keep it here.
But I added a sentence to the initial condition.

Reply:
Okay, that makes sense.